docs/install.md

```diff
@@ -174,15 +174,138 @@ dd if=xcp-ng-8.2.0-netinstall.iso of=/dev/sdX bs=8M oflag=direct
 
 Everything else is like the [regular install](install.md#start-the-host), except that it will not offer to install from local media, only from distant ones.
 
+## Network boot (PXE)
+
+### Requirements
+
+To get XCP-ng installed from PXE, you need:
+
+* DHCP and TFTP configured servers
+* Any NFS, FTP or HTTP server (your choice) to host XCP-ng installation files
+* A PXE-boot compatible network card on your host
+
+:::tip
+PXE boot doesn't support tagged VLAN networks! Be sure to boot on a untagged network!
+:::
+
+### TFTP server configuration
```

Comment:
The current structure seems a bit confusing to me (UEFI as an exception rather than clearly show BIOS vs UEFI, common files (install.img and such) put at different places when they could be in a single place if supporting both BIOS and UEFI installs, info related to XCP-ng installation files only present in the UEFI section...)

Here's a proposal:
```
### TFTP server configuration
#### Common files
(install.img), xen.gz, vmlinuz
#### BIOS boot
pxelinux and its config
#### UEFI boot
grub EFI and its config
### Installation files
NFS, FTP or HTTP server, with the tip about .treeinfo
### On the host
```
I think our grub supports HTTP so install.img could very well be downloader over HTTP, with the appropriate grub config. I'm not sure about pxelinux.0 though. I think I needed lxpelinux.0 to make it able to download over HTTP. HTTP would speed the boot up and allow not to duplicate install.img in the case when people choose HTTP to deliver the installer files.